### PR TITLE
docs(korean): added Korean

### PR DESCRIPTION
**Pull request by Jongjineee**
I added the Korean translation of this repo. Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
 [![version][version-badge]][package]
 [![MIT License][license-badge]][LICENSE]
 
-[![All Contributors](https://img.shields.io/badge/all_contributors-20-orange.svg?style=flat-square)](#contributors)
+[![All Contributors](https://img.shields.io/badge/all_contributors-21-orange.svg?style=flat-square)](#contributors)
```
Comment (sinchang) [inline README.md]: It seems that your user info has not been added to the readme.

Comment (Berkmann18) [reply README.md]: Looks like a manual change. Otherwise the table would have been updated accordingly.

Comment (Jongjineee) [reply README.md]: Thank you. I followed Berkmann18's PR and changed it manually. I'll fix it.

Comment (Berkmann18) [reply README.md]: @Jongjineee If you want to add yourself as a contributor, make sure you do it through the [`all-contributors-cli`](https://www.npmjs.com/package/all-contributors-cli) package.
In any case, the bot or us can do that.

Comment (Jongjineee) [reply README.md]: @Berkmann18 I want you to do it. So there's no problem with the code?

File: docs/ko-KR/README.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
+<div align="center">
+    <a href="https://github.com/all-contributors/all-contributors">
+        <img src="other/logo-full.png" alt="✨ All Contributors ✨" width="800px" />
```
Comment (jakebolam) [inline docs/ko-KR/README.md]: ../../other/full-logo.png

**Comment by jakebolam** [comment]
@Jongjineee are you interested in helping make the bot respond to languages other than English?

**Comment by Jongjineee** [comment]
@jakebolam I can speak Korean, can I do it?

File: docs/ko-KR/README.md (hunk 2 above)
Comment (Jongjineee) [reply docs/ko-KR/README.md]: Thank you. I fixed it!

**Comment by jakebolam** [comment]
@Jongjineee yes!

**Comment by jakebolam** [comment]
@allcontributors[bot] please add @Jongjineee for doc and translation

**Pull request merged**